packages/ui/components/validate-messages/src/loadDefaultFeedbackMessagesNoSideEffects.js

```diff
@@ -124,9 +129,10 @@ export function loadDefaultFeedbackMessagesNoSideEffects({ localize }) {
    * @returns {Promise<string|Element>}
    */
   const getLocalizedMessage = async data => {
+    const inputMode = data.inputMode !== 'enter' ? capitalize(data.inputMode) : '';
     await forMessagesToBeReady();
     if (data) {
-      return localize.msg(`lion-validate:${data.type}.${data.name}`, data);
+      return localize.msg(`lion-validate:${data.type}.${data.name}${inputMode}`, data);
```

Comment:
Should we consider a `_` inbetween here, to indicate it's a message variation for the same validator type?